test-apps/ui-test-app/src/frontend/appui/frontstages/LocalFileStage.tsx

```diff
@@ -140,6 +140,7 @@ class LocalFilePage extends React.Component<LocalFilePageProps, LocalFilePageSta
   };
 
   private _handleElectronFileOpen = async () => {
+    // NEEDSWORK: These OpenDialogOptions filters/extensions are not working

```

Comment:
what happens?

Reply:
All files in a folder are shown, and the filters are not listed in the drop-down in the dialog; only "All files (*.*)". I also tried the "title", which also did not work.

Reply:
let me look at it.

Reply:
fixed on this branch.

Reply:
Thanks